src/Statement/Grouper/StatementGrouper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Statement\Grouper;
+
+use Wikibase\DataModel\Statement\StatementList;
+
+/**
+ * @since 3.2
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+interface StatementGrouper {
+
+	/**
+	 * @param StatementList $statements
+	 *
+	 * @return StatementList[] An associative array, mapping statement group identifiers to
```

Comment:
Some further explanation of what "statement group identifiers" are would be nice